clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -97,17 +97,17 @@ public static synchronized TachyonFS get(final TachyonURI tachyonURI) throws IOE
    */
   public static synchronized TachyonFS get(final TachyonURI tachyonURI, TachyonConf tachyonConf)
       throws IOException {
-    Preconditions.checkNotNull(tachyonConf, "Could not pass null TachyonConf instance.");
-    if (tachyonURI == null) {
-      throw new IOException("Tachyon Uri cannot be null. Use " + Constants.HEADER + "host:port/ ,"
-          + Constants.HEADER_FT + "host:port/");
-    }
+    Preconditions.checkNotNull(tachyonConf, "Could not pass a null TachyonConf instance.");
+    Preconditions.checkNotNull(tachyonURI, "Tachyon URI cannot be null. Use " + Constants.HEADER
```

Comment:
These two checks can be checkArgument(), since they are checking the two method arguments.